### DISCUSSION TITLE
[Request] Config options to disable scrap and/or store items from spawning/being in the store

### DISCUSSION THREAD

**Discussion by MathiusXII** [EtherealEmil/UsualScrap#15] [Ideas]
For store items, could you still keep them in the game when they're disabled? I'd like to add them as Remnants items but I don't want them in the store.

(Remnants is a mod)

**Comment by EtherealEmil** [comment]
I don't know how I would do this as the items are registered as store items or scrap items in code and cease to exist if I don't choose one. Maybe I could have an option that sets them as scrap items but sets their spawn rate to 0 so you have to implement them yourself? Would that work? I don't really know how remnants works.

**Comment by MathiusXII** [comment]
I know LethalThings lets you remove items from the store, but they can still be spawned in with cheats so they're still in the game. This allows remnants to still add them as remnant items. If the items are removed from the game entirely that won't work.

**Comment by MathiusXII** [comment]
Btw remnants basically allows store items to spawn inside with a custom spawning system. It actually makes a duplicate version of all "blue equipment" as "green scrap" so they have value.

**Comment by EtherealEmil** [comment]
Ok, I think I understand what you're asking for. You're asking for an option to disable the scrap items from showing up naturally via the store or scrap spawn but they are still registered with the game which means they can be spawned via dev mode and thus used with the remnants mod?

I'll just point out this may not be added as soon as you'd like but it will have been written it in my things to implement, just need to understand exactly what I'd need to implement from a coding standpoint.

**Comment by MathiusXII** [comment]
No problem. I'll just keep your mod on my radar for whenever that gets added.

**Comment by EtherealEmil** [comment]
I know it's been a long while but this has now been added if you're still interested, sorry for the wait.
